File: Java/프로그래머스/0/120843. 공 던지기/공 던지기.java
class Solution {
    public int solution(int[] numbers, int k) {
        int answer = 0;
        int sum=0;
        int index=0;
        for(int i=0 ; i<k; i++){
            sum+=2;
        }
        int arr[] = new int[sum];
        for(int i=0; i<arr.length ; i++){
            arr[i]=numbers[i%numbers.length];
        }
        for(int i=0; i<k-1; i++){
            index+=2;
        }
        answer = arr[index];
        return answer;
    }
}

File: 프로그래머스/0/120843. 공 던지기/공 던지기.js
function solution(numbers, k) {
        let answer = 0;
        let sum=k*2;
        let index=sum-2;
        let arr = new Array(sum).fill(0);
        for(let i=0; i<arr.length ; i++){
            arr[i]=numbers[i%numbers.length];
        }
        answer = arr[index];
    return answer;
}